include/lbann/layers/learning/deconvolution.hpp

```diff
@@ -36,7 +36,9 @@ namespace lbann {
 // Forward declaration.
 class lbann_callback_imcomm;
 
-/// Deconvolution layer
+/**
+ * @brief Transpose of the convolution layer.
```

Comment:
I think transposed convolution is odd enough that it's worth having a bit more detail here.

Unrelated note: Perhaps we should rename this to "transposed convolution"?

Reply:
I would say we should stick with the deep-learningism here. If we wanted to be pedantic, convolution should actually be called "cross correlation".

Reply:
Both PyTorch and Tensorflow refer to it as transposed convolution, so I think that's become the standard.